core/src/test/java/io/aiven/klaw/service/EnvsClustersTenantsControllerServiceTest.java

```diff
@@ -0,0 +1,135 @@
+package io.aiven.klaw.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+import io.aiven.klaw.config.ManageDatabase;
+import io.aiven.klaw.dao.Env;
+import io.aiven.klaw.dao.EnvID;
+import io.aiven.klaw.dao.UserInfo;
+import io.aiven.klaw.error.KlawException;
+import io.aiven.klaw.error.KlawValidationException;
+import io.aiven.klaw.helpers.db.rdbms.HandleDbRequestsJdbc;
+import io.aiven.klaw.model.ApiResponse;
+import io.aiven.klaw.model.EnvModel;
+import io.aiven.klaw.model.enums.ApiResultStatus;
+import io.aiven.klaw.model.enums.KafkaClustersType;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.util.ReflectionTestUtils;
+
+@ExtendWith(SpringExtension.class)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+class EnvsClustersTenantsControllerServiceTest {
+
+  private EnvsClustersTenantsControllerService service;
```

Comment:
Can we add couple of tests for schemas here https://github.com/aiven/klaw/blob/main/core/src/test/java/io/aiven/klaw/EnvsClustersTenantsControllerIT.java#L278?